pkg/v1/providers/infrastructure-azure/v1.0.0/ytt/overlay.yaml

```diff
@@ -301,6 +301,9 @@ spec:
 apiVersion: v1
 kind: Secret
 metadata:
+  metadata:
```

Comment:
@prkalle Is adding `metadata` here intentional?

Reply:
Thanks for catching this, copy paste error.